ExecDef/tools.py

```diff
@@ -0,0 +1,59 @@
+import json
+import re
+from pathlib import Path
+
+from gooddata_pandas import GoodPandas
+from gooddata_sdk import Attribute, ExecutionDefinition, GoodDataSdk, ObjId, SimpleMetric
+
+
+def get_workspace_id() -> str:
+    sdk = GoodDataSdk.create_from_profile(profile="default", profiles_path=Path("./profiles.yaml"))
+    return sdk.catalog_workspace.list_workspaces()[0].workspace_id
```

Comment:
Not sure about this. Why do you pick the first workspace from the list of workspaces?

Reply:
Just for simplicity, as this presumes, that the user will only have demo as their only workspace.